examples/long_running.py

```diff
@@ -9,6 +11,15 @@
 # ======================================================================================
 
 
+# --------------------------------------------------------------------------------------
+# Categorical variables
+# --------------------------------------------------------------------------------------
+@dataclass
+class WorkingState:
```

Comment:
```suggestion
class Working:
```
Both a state and a choice, so avoid `State` for sure.